plugins/janus_videoroom.c

```diff
@@ -558,7 +559,7 @@ typedef struct janus_videoroom {
 	gint64 destroyed;			/* Value to flag the room for destruction, done lazily */
 	GHashTable *participants;	/* Map of potential publishers (we get listeners from them) */
 	GHashTable *private_ids;	/* Map of existing private IDs */
-	gboolean check_tokens;		/* Whether to check tokens when participants join (see below) */
+	gboolean check_allowed;		/* Whether to check tokens when participants join (see below) */
```

Comment:
Is this replacing the existing ACL mechanism we have, or just renaming the property?

If it's the former, I'm hesitant to simply replace the ACL we have right now with this new token mechanism: this would not be backwards compatible, and would break the applications of those using the existing ACL right now. I'd rather add this as a new, alternative, mechanism that people can start making use of. We can then make it clearer, when merging, that the other approach will be deprecated soon, and that people should start migrating to this ASAP. Of course, if that's not happening, feel free to disregard this note...

Reply:
I renamed this property to make it a bit less confusing since this property has nothing to do with the signed tokens in this PR.

Reply:
Ack, thx :+1: